iris_grib/tests/unit/save_rules/test_set_discipline_and_parameter.py

```diff
@@ -0,0 +1,88 @@
+# (C) British Crown Copyright 2019, Met Office
+#
+# This file is part of iris-grib.
+#
+# iris-grib is free software: you can redistribute it and/or modify it under
+# the terms of the GNU Lesser General Public License as published by the
+# Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# iris-grib is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with iris-grib.  If not, see <http://www.gnu.org/licenses/>.
```

Comment:
You should also update this license header

Reply:
Oops.
Let's see if the new header check finds it !

Reply:
yup! :1st_place_medal: